docs/metrics/exemplars/Program.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using OpenTelemetry;
+using OpenTelemetry.Exporter;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+public class Program
+{
+    private static readonly ActivitySource MyActivitySource = new("OpenTelemetry.Demo.Exemplar");
+    private static readonly Meter MyMeter = new("OpenTelemetry.Demo.Exemplar");
+    private static readonly Counter<long> MyFruitCounter = MyMeter.CreateCounter<long>("MyFruitCounter");
```

Comment:
I'd suggest to use Histogram for the Exemplar demo as that is an instrument where Exemplars are commonly used.

Reply:
Updated to histogram.